content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-upgrade.md

```diff
@@ -179,14 +178,11 @@ Also calling `kubeadm upgrade plan` and upgrading the CNI provider plugin is no
     # since apt-get version 1.1 you can also use the following method
     apt-get update && \
     apt-get install -y --allow-change-held-packages kubelet={{< skew currentVersion >}}.x-00 kubectl={{< skew currentVersion >}}.x-00
-    </pre>
-{{< /tab >}}
-{{< tab name="CentOS, RHEL or Fedora" >}}
-    <pre>
+{{% /tab %}}
+{{% tab name="CentOS, RHEL or Fedora" %}}
     # replace x in {{< skew currentVersion >}}.x-0 with the latest patch version
     yum install -y kubelet-{{< skew currentVersion >}}.x-0 kubectl-{{< skew currentVersion >}}.x-0 --disableexcludes=kubernetes
-    </pre>
-{{< /tab >}}
+{{% /tab %}}
 {{< /tabs >}}
 
```

Comment:
Would it be possible to add a new line here right before the `Restart the kubelet` section begins? 

![image](https://user-images.githubusercontent.com/56963264/133994645-e50dc89b-ff3c-46dd-a012-14635da91a12.png)
Looks a bit cramped currently 😅

Reply:
I totally agree! 😅 However, I already tried that in https://github.com/kubernetes/website/pull/29748/commits/cc8bda3f410cfe51f2d04060a6e1da2389b24f98, and it didn't seem to work 🤔 Do you have a different suggestion to where I could add the newline?

Reply:
I think `ENTER` for new line in markdown works only once. If you want to force a line break using `<br />` might work, could you please try that?